src/main/java/net/neoforged/neoforge/common/extensions/IEntityExtension.java

```diff
@@ -135,24 +149,42 @@ default MobCategory getClassification(boolean forSpawnCount) {
      *
      * @return True if this entity is being tracked by a world
      */
-    // TODO: rename in 1.19 to isAddedToLevel
-    boolean isAddedToWorld();
+    boolean isAddedToLevel();
 
     /**
-     * Called after the entity has been added to the world's
-     * ticking list. Can be overriden, but needs to call super
-     * to prevent MC-136995.
+     * @deprecated Use {@link #isAddedToLevel()} instead.
+     * @return True if this entity is being tracked by a world
      */
-    // TODO: rename in 1.19 to onAddedToLevel
-    void onAddedToWorld();
+    @Deprecated(forRemoval = true, since = "21.0")
+    default boolean isAddedToWorld() {
+        return isAddedToLevel();
+    }
 
     /**
-     * Called after the entity has been removed to the world's
-     * ticking list. Can be overriden, but needs to call super
-     * to prevent MC-136995.
+     * Called after the entity has been added to the world's ticking list.
      */
-    // TODO: rename in 1.19 to onRemovedFromLevel
-    void onRemovedFromWorld();
+    void onAddedToLevel();
+
+    /**
+     * @deprecated Use {@link #onAddedToLevel()} instead.
+     */
+    @Deprecated(forRemoval = true, since = "21.0")
+    default void onAddedToWorld() {
```

Comment:
Does this really need to stay around as baggage? NeoForge's still in the breaking-changes phase of 1.21.